### PR TITLE
Add scripts for mysql and docker-compose file for starting Grafana and MySQL

### PR DESCRIPTION
**Pull request by Kozl0ff**

### CODE REVIEW

File: docker/micro-cluster-lab/scripts_db/mysql/init/test_create.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Create schema nk;
+Create table test(id INT NOT NULL, name Varchar NOT NULL, value INT NOT NULL);
```
Comment (S4RPYK3IVl5) [inline docker/micro-cluster-lab/scripts_db/mysql/init/test_create.sql]: Do you need this test script?
If do, then:
Rename schema to more common name.
Write SQL statements in upper case.

**Review by S4RPYK3IVl5 — CHANGES_REQUESTED** [review]

File: docker/micro-cluster-lab/mysql/init/test_create.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Create table test(id INT NOT NULL, value INT NOT NULL, data DATE NOT NULL);
```
Comment (S4RPYK3IVl5) [inline docker/micro-cluster-lab/mysql/init/test_create.sql]: Change **Create table** to **CREATE TABLE**

Comment (Kozl0ff) [reply docker/micro-cluster-lab/mysql/init/test_create.sql]: ok

File: docker/micro-cluster-lab/scripts_db/mysql/init/test_create.sql (hunk 1 above)
Comment (Kozl0ff) [reply docker/micro-cluster-lab/scripts_db/mysql/init/test_create.sql]: I change this script

**Review by S4RPYK3IVl5 — APPROVED** [review]

**Pull request merged**